src/main/webapp/app/guided-tour/guided-tour-step.model.ts

```diff
@@ -31,12 +13,58 @@ export interface TourStep {
     skipStep?: boolean;
```

Comment:
```suggestion
    skipStep = false; //default value that needs to be overridden if a tour step should be skipped 
```

Reply:
the comment above the attribute is bad English, please improve

Reply:
@krusche I have updated the comments